tests/expected/cpp/60087-indent_namespace_single_indent.cpp

```diff
@@ -0,0 +1,8 @@
+namespace out
+{
+	int i;
+namespace in
```

Comment:
This does not seem right. The existing 'indent_namespace_single_indent' is used to control the indentation when multiple consecutive namespaces are found. If the parameter was true, the first namespace would be indented as per current indent level, while subsequent namespaces would use the same indent level as the first. 
You can look at the test file '30914-indent_namespace_single_indent.h' (and relative input and config) to see how that parameter works.
If you run this example with the config file 'indent_namespace_single_indent.cfg', you will get this output: 
```
namespace out
{
	int i;
        namespace in
        {
	        int i;
        }
}
```

I guess what you wanted to do is a bit different, that is controlling whether to indent the code inside a namespace or not. Problably wiser to use a new separate option for that.